cypress/utils/StaffPage/StaffPageTestFunctions.ts

```diff
@@ -27,4 +59,176 @@ export class StaffPageTestFunctions {
     this.getStaffTile().click();
     cy.url().should('include', '/staff');
   };
+
+  testStaffList = () => {
+    this.mockGETStaff();
+    this.getStaffListTile().should('exist');
+    this.getStaffListTile().should('have.length', 5);
+    this.getStaffListTile().eq(0).contains('dr Joanna Czarnowska');
+    this.getStaffListTile()
+      .eq(0)
+      .contains('Wydział Matematyki, Fizyki i Informatyki • Instytut Informatyki • Zakład Sztucznej Inteligencji');
+    this.getStaffListTile().eq(1).contains('dr Mikołaj Czechlewski');
+    this.getStaffListTile()
+      .eq(1)
+      .contains('Wydział Matematyki, Fizyki i Informatyki • Instytut Informatyki • Zakład Języków Formalnych');
+    this.getStaffListTile().eq(2).contains('dr Hanna Furmańczyk');
+    this.getStaffListTile()
+      .eq(2)
+      .contains(
+        'Wydział Matematyki, Fizyki i Informatyki • Instytut Informatyki • Zakład Optymalizacji Kombinatorycznej',
+      );
+    this.getStaffListTile().eq(3).contains('mgr Mateusz Miotk');
+    this.getStaffListTile()
+      .eq(3)
+      .contains(
+        'Wydział Matematyki, Fizyki i Informatyki • Instytut Informatyki • Zakład Optymalizacji Kombinatorycznej',
+      );
+    this.getStaffListTile().eq(4).contains('dr hab. Paweł Żyliński, prof. UG');
+    this.getStaffListTile()
+      .eq(4)
+      .contains(
+        'Wydział Matematyki, Fizyki i Informatyki • Instytut Informatyki • Zakład Optymalizacji Kombinatorycznej',
+      );
+  };
+
+  testStaffLink = () => {
+    this.mockGETStaff();
+    this.getStaffListContainer().should('exist');
+    this.getStaffListTileLink().eq(0).should('have.attr', 'href', '/staff/63cb1cf20ada513d831bc83d');
+    this.getStaffListTileLink().eq(1).should('have.attr', 'href', '/staff/63cb1cf20ada513d831bc83f');
+    this.getStaffListTileLink().eq(2).should('have.attr', 'href', '/staff/63cb1cf20ada513d831bc85c');
+    this.getStaffListTileLink().eq(3).should('have.attr', 'href', '/staff/63cb1cf20ada513d831bc8c9');
+    this.getStaffListTileLink().eq(4).should('have.attr', 'href', '/staff/63cb1cf20ada513d831bc956');
+
+    this.mockGETStaffDetails();
+    this.getStaffListTileLink().eq(0).click();
+    cy.location().should(loc => {
+      expect(loc.pathname).to.eq('/staff/63cb1cf20ada513d831bc83d');
+    });
+    cy.go('back');
+    this.mockGETStaffDetails2();
+    this.getStaffListTileLink().eq(3).click();
+    cy.location().should(loc => {
+      expect(loc.pathname).to.eq('/staff/63cb1cf20ada513d831bc8c9');
+    });
```

Comment:
I think that this part can be deleted since you already test the href attribute right above